src/interfaces/chain.h

```diff
@@ -154,7 +154,10 @@ class Chain
     //! Transaction is added to memory pool, if the transaction fee is below the
     //! amount specified by max_tx_fee, and broadcast to all peers if relay is set to true.
     //! Return false if the transaction could not be added due to the fee or for another reason.
-    virtual bool broadcastTransaction(const CTransactionRef& tx, std::string& err_string, const CAmount& max_tx_fee, bool relay) = 0;
+    virtual bool broadcastTransaction(const CTransactionRef& tx,
+        const CAmount& max_tx_fee,
+        bool relay,
+        std::string& err_string) = 0;
```

Comment:
nit: is this needed to break the line here while keeping this function definition one-liner in chain.cpp?

Reply:
re: https://github.com/bitcoin/bitcoin/pull/18278#discussion_r389315317

> nit: is this needed to break the line here while keeping this function definition one-liner in chain.cpp?

added same wrapping